develop/reference/articles/xx-theme-file-reference/00-theme-reference-intro.markdown

```diff
@@ -0,0 +1,241 @@
+which files to modify and which files to leave untouched.
+
+## Theme Anatomy [](id=theme-anatomy)
+
+There are two main approaches to theme development for @product@ 7: themes built 
+using the Node build tools with the [themes generator](/develop/tutorials/-/knowledge_base/7-0/themes-generator) 
+and themes built using the Plugins SDK.
+
+For themes developed with the themes generator, your theme anatomy is organized
+with the structure shown below. Although themes developed with the Plugins SDK 
+have a slightly different anatomy, the core theme files are the same. Note that 
+the `build` directory is shown for reference, and is not created by default.
+
+- `theme-name/`
+    - `build/`(not created automatically)
+        - `css/`
+            - [`_application.scss`](#application-scss)
+            - [`_aui_custom.scss`](#auicustom-scss)
+            - [`_aui_variables.scss`](#auivariables-scss)
+            - [`_base.scss`](#base-scss)
+            - [`_custom.scss`](#custom-scss)
+            - [`_extras.scss`](#extras-scss)
+            - [`_imports.scss`](#imports-scss)
+            - [`_layout.scss`](#layout-scss)
+            - [`_liferay_custom.scss`](#liferaycustom-scss)
+            - [`_liferay_variables_custom.scss`](#liferayvariablescustom-scss)
+            - [`_liferay_variables.scss`](#liferayvariables-scss)
+            - [`_navigation.scss`](#navigation-scss)
+            - [`_portal.scss`](#portal-scss)
+            - [`_portlet.scss`](#portlet-scss)
+            - [`_taglib.scss`](#taglib-scss)
+            - `application/`
+            - `aui/`
+            - [`aui.scss`](#aui-scss)
+            - `base/`
+            - `layout/`
+            - [`main.scss`](#main-scss)
+            - `navigation/`
+            - `portal/`
+            - `portlet/`
+            - `taglib/`
+        - `templates/`
+            - [`init_custom.ftl`](#initcustom-ftl)
+            - [`init.ftl`](#init-ftl)
+            - [`navigation.ftl`](#navigation-ftl)
+            - [`portal_normal.ftl`](#portalnormal-ftl)
+            - [`portal_pop_up.ftl`](#portalpopup-ftl)
+            - [`portlet.ftl`](#portlet-ftl)
+            - (other directories that have been copied from src)
+    - `dist/` (not created automatically)
+    - [`gulpfile.js`](#gulpfile-js)
+    - [`liferay-theme.json`](#liferay-theme-json)
+    - `node_modules/`
+        - (many directories)
+    - [`package.json`](#package-json)
+    - `src/`
+        - `css/`
+            - (modified CSS files)
+        - `images/`
+            -   (many directories)
+        - `js/`
+            - [`main.js`](#main-js)
+        - `templates/`
+            - (Modified theme templates)
+        - `WEB-INF/`
+                - `lib/`
+            - [`liferay-look-and-feel.xml`](#liferay-look-and-feel-xml)
+            - [`liferay-plugin-package.properties`](#liferay-plugin-package-properties)
+            - `src/`
+                - `resources-importer/`
```

Comment:
We're going to remove this directory from the generated themes soon, and add a task that templates out the resources importer files. Mostly because it's not actually essential to a theme. Up to you if you want to leave it here or not, just a heads up.